node/primitives/src/currency.rs

```diff
@@ -221,6 +226,7 @@ macro_rules! create_currency_id {
 					CurrencyId::LPToken(_ts1, _, _ts2, _) => {
 						stringify!(_ts1, ",", _ts2)
 					}
+					_ => "Unknown",
```

Comment:
Is it possible that we maintain a map between Erc20 token list and bifrost TokenSymbol list? Might it be easier to see and convert?